midi/midi/decode.py

```diff
@@ -197,12 +231,14 @@ def get_midi_length(file: MidiFile,
 
 def check_file_type(file: MidiFile):
     """
-    checks if MIDI file type is 2: if yes, raises a ValueError
+    checks if MIDI file type is 1: if not, raises a ValueError
 
     :param file:
     :return:
     """
-    if file.type == 2:
+    if file.type == 0:
+        raise ValueError('impossible to perform calculations for type 0 (single-track) file')
```

Comment:
What does this entail? If someone uploads a midi file which has a single track the decoder will just fail?

Reply:
yes, tempo changes and notes track must be separated